js/app.jsx

```diff
@@ -517,6 +536,7 @@ var Job = React.createClass({
       ['State', state],
       ['Input', inputs],
       ['Output', cleanJobPath(job.conf.output)],
+      ['Scalding Steps', steps],
```

Comment:
From the user's perspective isn't this showing something more like line numbers?